tee-worker/omni-executor/aa-contracts/src/accounts/OmniAccount.sol

```diff
@@ -32,9 +32,19 @@ contract OmniAccount is BaseAccount, TokenCallbackHandler, UUPSUpgradeable, Init
 
     mapping(address => bool) public rootSigners;
     mapping(bytes32 => bool) public passkeySigners;
+    uint256 public passkeySignerCount;
 
     IEntryPoint private immutable _entryPoint;
 
+    // Selectors for restricted functions that only owner should call
+    bytes4 private constant ADD_ROOT_SIGNER_SELECTOR = bytes4(keccak256("addRootSigner(address)"));
+    bytes4 private constant REMOVE_ROOT_SIGNER_SELECTOR = bytes4(keccak256("removeRootSigner(address)"));
+    bytes4 private constant ADD_PASSKEY_SIGNER_SELECTOR = bytes4(keccak256("addPasskeySigner((uint256,uint256))"));
+    bytes4 private constant REMOVE_PASSKEY_SIGNER_SELECTOR = bytes4(keccak256("removePasskeySigner((uint256,uint256))"));
```

Comment:
Should it be flattened fn sig like:
- `addPasskeySigner(uint256,uint256)`
- `removePasskeySigner(uint256,uint256)`
? 

We'll need tests for it

Reply:
I see - thanks for the explanation